index.css

```diff
@@ -414,3 +414,155 @@ hr {
   margin-bottom: 20px;
   padding: 0;
 }
+
+.nav-bar-desk {
+  display: none;
+}
+
+/* Media queries for desktop view */
+
+@media screen and (min-width: 768px) {
+  .nav-bar-desk {
+    display: flex;
+  }
+
+  .main-section {
+    display: flex;
+    flex-direction: column;
+    margin: 5% 260px 325px;
+  }
+
+  .three-bar-menu {
+    display: none !important;
+  }
+
+  body {
+    background-size: 95%;
+    background-position-y: 60px;
+    background-position-x: 34.6px;
+  }
+
+  .header-bar {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+  }
+
+  .header-bar header {
+    padding-left: 100px;
+  }
+
+  .nav-bar-list {
+    display: flex;
+    justify-content: space-between;
+  }
+
+  .ul-desk {
+    display: flex;
+  }
+
+  li {
+    padding: 0 20px;
+  }
+
+  li a {
+    color: #344563;
+    font-weight: 500;
+    font-size: 15px;
+  }
+
+  li:last-child {
+    padding-right: 12px;
+  }
+
+  .cards {
+    flex-direction: row;
+    height: 452px;
+    margin: 114px 8% 15%;
+  }
+
+  .cards .second-container {
+    margin-left: 0;
+    margin-right: 0;
+  }
+
+  .cards .second-container p {
+    max-width: 100%;
+  }
+
+  .main-img img {
+    width: 465px;
+    height: 400px;
+    margin: 0 14px 0;
+  }
+
+  .cards article .horizontal-section {
+    justify-content: flex-start;
+  }
+
+  .horizontal-section > * {
+    padding-right: 10px;
+    margin-left: 0;
+  }
+
+  .card-2 .main-img,
+  .card-4 .main-img {
+    order: 1;
+  }
```

Comment:
- [ ] Kindly increase the padding around the project cards in the `mobile version` so that there is enough spacing around elements like in the original design.

![image](https://user-images.githubusercontent.com/98453979/220969888-2aafa079-8b43-4df9-abeb-6349ac95d90f.png)

### Checkout the original design

![image](https://user-images.githubusercontent.com/98453979/220970129-352f8bb1-6fb1-415b-a313-fa5f183af4c4.png)